default/techs.txt

```diff
@@ -2276,7 +2291,7 @@ Tech
     researchturns = 8
     prerequisites = [
         "SHP_MULTICELL_CAST"
-        "GRO_LIFECYCLE_MAN"
+        "SHP_MONSTER_MATURATION"
```

Comment:
I'm not really understanding the reasoning for adding this as a prereq for SHP_ENDOCRINE_SYSTEMS -- it's just that there was a long connecting line in the tech tree display?

If you simply make this another tech that follows GRO_LIFECYCLE_MAN, but which isn't actually a prereq to anything, then there is no urgent need for any AI adaptation to this-- they don't really have such nuanced monster handling yet for me to be concerned about giving them extra monster-wrangler skills right away.